src/main.js

```diff
@@ -134,14 +181,25 @@ export class Draggable {
                 unbind(this.document, "pointercancel", this._pointerup);
                 unbind(this.document, "contextmenu", preventDefault);
 
+                if (this.clickMoveClick){
+                    unbind(this.document, "keydown", this._keydown);
```

Comment:
Do we need to unbind other events, e.g.:

https://github.com/telerik/kendo-draggable/pull/53/files#diff-27653c212e1cfe533e4eb2f7d0d3f89604c9de48a09583b4cbbbcbd08a07da79R152